src/gcodeExport.cpp

```diff
@@ -711,8 +727,18 @@ void GCodeExport::writeUnretractionAndPrime()
         }
         else
         {
-            current_e_value += extruder_attr[current_extruder].retraction_e_amount_current;
-            *output_stream << "G1 F" << PrecisionedDouble{1, extruder_attr[current_extruder].last_retraction_prime_speed * 60} << " " << extruder_attr[current_extruder].extruderCharacter << PrecisionedDouble{5, current_e_value} << new_line;
+            double output_e;
```

Comment:
It can be so much simpler.
Like this:
```
current_e_value += extruder_attr[current_extruder].retraction_e_amount_current;
double output_e = (relative_extrusion)? extruder_attr[current_extruder].retraction_e_amount_current + prime_volume_e : current_e_value;
```